docs/make.jl

```diff
@@ -17,7 +17,7 @@ makedocs(sitename = "ColPrac",
          ],
          format = Documenter.HTML(analytics = "UA-90474609-3",
                                   assets = ["assets/favicon.ico"],
-                                  canonical = "https://colprac.sciml.ai/stable/"),
+                                  canonical = "https://docs.sciml.ai/ColPrac/stable/"),
```

Comment:
likewise, 403 Forbdden

Reply:
Yep, this is the same key issue. As above Chris said to point toward stable, and the key issue will be resolved separately. I did check and the `dev` version of this link works, so it is just a matter of resolving that key issue.

Reply:
cool, let's wait til that's fixed and the link works before merging

Reply:
Sure, that sounds fine. No problem.

Reply:
thanks! 

I see the old link is dead, so thanks for getting this updated (still good to make sure the new one is working before the merge) :)

Reply:
We need to solve one problem at a time. If we put the wrong link it'll never get corrected. Instead, if you could collect the names of the packages that need a stable and put them in SciMLDocs, we can go fix that one by one.